hack/prepare-ci.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+
+# Copyright 2021 The Crossplane Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -euo pipefail
+
+PROVIDER_NAME="$1"
+find .github -type f -exec sed -i '' -e "s/{{Template}}/${PROVIDER_NAME}/g" {} \;
```

Comment:
Not sure if we really need a separate script. 
Maybe we can just add this as yet another command in step 3 here: https://github.com/crossplane-contrib/terrajet/blob/main/docs/generating-a-provider.md#generate

Reply:
I'm fine both ways. We may add the command in the guide and if more is needed, collect them in a script afterwards.